src/lib/autoinstall/clients/inst_autosetup.rb

```diff
@@ -88,6 +89,7 @@ def main
           _("Configuring Systemd Default Target..."),
           _("Importing users and groups configuration..."),
           _("Importing SSH keys/settings..."),
+          _("Writing configuration files..."),
```

Comment:
It is a bit strange to see it twice. Cannot we get title from that finish client info call?

Reply:
It is strange because it is simply wrong. :-) we are reading the configuration files here only.